src/robot/client.ts

```diff
@@ -741,6 +744,7 @@ export class RobotClient extends EventDispatcher implements Robot {
     request.setEvery(new Duration().setNanos(durationMs * 1e6));
 
     const statusStream = robotService.streamStatus(request);
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Warn rather than suppress, unless this is to prevent an auto fix, in which case can you add a TODO comment to go along with the suppression?